src/bsp/platform/posix/xi_bsp_io_fs_posix.c

```diff
@@ -236,15 +243,17 @@ xi_bsp_io_fs_state_t xi_bsp_io_fs_read( const xi_bsp_io_fs_resource_handle_t res
     *buffer      = NULL;
     *buffer_size = 0;
     xi_bsp_mem_free( elem->memory_buffer );
+    elem->memory_buffer = NULL;
```

Comment:
ATTENTION HERE: I think this is the only relevant change in this file, others all are code formatter changes.

Reply:
I'm very confused about the formatting.  Perhaps we have different style definitions which is why this keeps happening?  I execute the tools/xi_format_code.py function should "should" pickup the .clang_format file in the base directory.

Reply:
I use sublime text 3's clang formatter extension with settings... checking

Reply:
and yes, I am confused too with the formatter differences between our computers

Reply:
atigyi$ clang-format --version
clang-format version 4.0.0 (tags/google/testing/2016-08-03)

Reply:
```
$ clang-format --version
clang-format version 5.0.0 (tags/google/stable/2017-06-22)
```

for reference